test/test_utils.py

```diff
@@ -120,8 +123,11 @@ def test_draw_boxes(self):
             res = Image.fromarray(result.permute(1, 2, 0).contiguous().numpy())
             res.save(path)
 
-        expected = torch.as_tensor(np.array(Image.open(path))).permute(2, 0, 1)
-        self.assertTrue(torch.equal(result, expected))
+        if PILLOW_VERSION >= (8, 2):
```

Comment:
Another solution is to set the condition to `if PILLOW_VERSION < (8, 2)` and not update the image.